scripts/docker/zephyrrc

```diff
@@ -0,0 +1,2 @@
+export ZEPHYR_SDK_INSTALL_DIR=/opt/sdk/zephyr-sdk-0.9.5/
```

Comment:
The content of this file can be set as ```ENV``` variables in the docker file, this will reduce the number of locations where the SDK version needs to be updated.